google-account-plugin/src/com/google/cloud/tools/intellij/login/IntellijGoogleLoginService.java

```diff
@@ -83,6 +83,37 @@ private IntellijGoogleLoginService() {
     addLoginListenersFromExtensionPoints();
   }
 
+  @Nullable
+  private static Project getCurrentProject() {
+    Window activeWindow = WindowManagerEx.getInstanceEx().getMostRecentFocusedWindow();
+    if (activeWindow == null) {
+      return null;
```

Comment:
Ah, I see this code was already existing. I leave it to you if you want to make this change or not.